shared/logger.go

```diff
@@ -90,9 +90,15 @@ func NewAppEngineContext(r *http.Request) context.Context {
 type gcLogger struct {
 	childLogger *gclog.Logger
 	traceID     string
+	maxSeverity gclog.Severity
 }
 
 func (gcl *gcLogger) log(severity gclog.Severity, format string, params ...interface{}) {
+	// The underlying type of gclog.Severity is int, https://pkg.go.dev/cloud.google.com/go/logging#Severity.
```

Comment:
The more important piece of doc is about the order:
```suggestion
	// "Severity levels are ordered, with numerically smaller levels treated as less severe than numerically larger levels".
	// https://pkg.go.dev/cloud.google.com/go/logging#Severity.
```

Reply:
Thanks!